docs/en/index.md

```diff
@@ -18,7 +18,7 @@ FreshRSS has a lot of features including:
 
 This documentation is split into different sections:
 
-* [User documentation](./users/02_First_steps.html), where you can discover all the possibilities offered by FreshRSS
-* [Administrator documentation](./admins/01_Index.html) for detailed installation and maintenance related tasks
-* [Developer documentation](./developers/01_First_steps.html) to guide you in the source code of FreshRSS and to help you if you want to contribute
+* [User documentation](./users/02_First_steps.md), where you can discover all the possibilities offered by FreshRSS
```

Comment:
You can't do that. ;-)

https://freshrss.github.io/FreshRSS/en/users/02_First_steps.html
https://freshrss.github.io/FreshRSS/en/users/02_First_steps.md

Reply:
Github Pages' uses the jekyll-relative-links plugin to convert relative ".md" links to ".html". Should be good to go! 

See:
https://github.com/FreshRSS/FreshRSS/blob/master/docs/en/admins/01_Index.md
https://freshrss.github.io/FreshRSS/en/admins/01_Index.html